torch/quantization/fx/utils.py

```diff
@@ -339,7 +340,7 @@ def create_qparam_nodes(quantizer: QuantizerCls, node_name: str, scale: Any, zer
     return (scale_node, zero_point_node)
 
 
-def all_node_args_have_no_tensors(node: Node) -> bool:
+def all_node_args_have_no_tensors(node: Node, modules: Dict[str, torch.nn.Module]) -> bool:
```

Comment:
would it be possible to pass around the parent module instead, and look up the module dynamically every time?  This avoids the need for one more custom stateful container.  I think the performance concern is ok (this shouldn't be the bottleneck) if we get lower code complexity in return.

Reply:
not sure I understand, can you clarify with code?

Reply:
oh, it's the same thing from the above comment, your suggestion of doing `getattr(model, node.target)` sounds great, separate PR sounds great